src/CONST.js

```diff
@@ -274,6 +279,7 @@ const CONST = {
                 [PLATFORM_OS_MACOS]: {input: 'j', modifierFlags: keyModifierCommand},
                 [PLATFORM_IOS]: {input: 'j', modifierFlags: keyModifierCommand},
             },
+            type: '',
```

Comment:
I'm not sure that we want `type: ''` instead of just not defining the key. I think I prefer leaving the key undefined.

Are there other constants where we prefer to define keys with empty values?